regtests/ww3_tp2.22/input/ww3_ounf.inp

```diff
@@ -0,0 +1,43 @@
+$ -------------------------------------------------------------------- $
+$ WAVEWATCH III Grid output post-processing                            $
+$--------------------------------------------------------------------- $
+$ First output time (yyyymmdd hhmmss), increment of output (s),
+$ and number of output times.
+$
+  19680606 000000 3600. 12
+$
+$ Fields requested --------------------------------------------------- $
+$
+$ Output request flags identifying fields as in ww3_shel.inp. See that
+$ file for a full documentation of field output options. Namelist type
+$ selection is used here (for alternative F/T flags, see ww3_shel.inp).
+$
+ N
+ HS T01 DIR SPR USS USSH
+$
+$--------------------------------------------------------------------- $
+$ netCDF version [3,4]
+$        and variable type 4 [2 = SHORT, 3 = it depends , 4 = REAL]
+$ swell partitions [0 1 2 3 4 5]
+$ variables in same file [T] or not [F]
+$
+ 3 4
```

Comment:
Since this is a new test, should we use NetCDF v4 or is there a particular reason to use v3?

Reply:
I used the `tp2.1` `ww3_grid.inp` file which had that setting.  I'll update that.  I also just realized I documented that `tp2.2` was used as a template for  `tp2.22`, though it was actually `tp2.1`, and `tp2.2` does not exist. I'll update any documentation that has that.

Reply:
fyi, `tp2.2` does exist, my mistake.

Reply:
Since this is a new test and we're making other changes, let's go ahead and move to NetCDF v4 instead of 3.

Reply:
This change has already been made.

Reply:
Apologies, it didn't show up correctly when I looked a minute ago.  I see that it's changed now.  Thank you!!!